app/uk/gov/hmrc/individualsifapistub/services/EmploymentsService.scala

```diff
@@ -40,4 +50,18 @@ class EmploymentsService @Inject()(employmentsRepository: EmploymentRepository)
           fields: Option[String]): Future[Option[Employments]] = {
     employmentsRepository.findByIdAndType(idType, idValue, startDate, endDate, fields)
   }
+
+  def verifyNino(idType: String, idValue: String)
```

Comment:
Possible to move this into a common place? The logic seems to be duplicated throughout services, could be case for a common base service to inherit from?